planner/core/integration_test.go

```diff
@@ -5294,6 +5294,42 @@ func TestIssue29503(t *testing.T) {
 	require.Len(t, res.Rows(), 2)
 }
 
+func TestIndexJoinCost(t *testing.T) {
```

Comment:
I ran this test on Master locally and all IndexJoins have the same costs with this PR.